### DISCUSSION TITLE
Rank selection and sss selection are currently the same ?

### DISCUSSION THREAD

**Discussion by CharlesArnal** [ahmedfgad/GeneticAlgorithmPython#205] [General]
Hello,
I think that currently, the code for rank selection and the code for steady state selection are exactly the same, which shouldn't be the case based on the usual definitions.

Thanks for creating this great library!

**Comment by ahmedfgad** [comment]
Thanks @CharlesArnal. This is fixed and pushed to the GitHub repository. It will be available in the next release.
